validator/testcases/javascript/actions.py

```diff
@@ -595,7 +602,9 @@ def _expr_assignment(traverser, node):
     if node["operator"] == "=":
 
         global_overwrite = False
-        readonly_value = True
+        # FIXME(Kris): This is not a great heuristic.
+        from predefinedentities import is_shared_scope
+        readonly_value = is_shared_scope(traverser)
```

Comment:
This is the non-trivial change to the `readonly` part you were talking about? @mstriemer could you please have a look?